prepareShiny.wdl

```diff
@@ -0,0 +1,108 @@
+# SOFTWARE.
+
+task CreateDesignMatrix {
+    input {
+        File countTable
+        String shinyDir = "."
+
+        Int threads = 1
+        String memory = "5G"
+        Int timeMinutes = 30
+        String dockerImage = "quay.io/biocontainers/predex:0.9.1--pyh3252c3a_0"
+    }
+
+    command {
+        set -e
+        mkdir -p ${shinyDir}
+        predex design \
+            -i ${countTable} \
+            -o ${shinyDir}
+    }
+
+    output {
+        File dgeDesign = shinyDir + "/design_matrix.tsv"
+    }
+
+    runtime {
+        cpu: threads
+        memory: memory
+        time_minutes: timeMinutes
+        docker: dockerImage
+    }
+
+    parameter_meta {
+        countTable: {description: "The created count table from HTseq.", category: "required"}
+        shinyDir: {description: "The directory to write the output to.", category: "required"}
+        
+        threads: {description: "The number of threads to use.", category: "advanced"}
+        memory: {description: "The amount of memory this job will use.", category: "advanced"}
+        timeMinutes: {description: "The maximum amount of time the job will run in minutes.", category: "advanced"}
+        dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.",
+                      category: "advanced"}
+    }
+}
+
+task CreateAnnotation {
+    input {
+        File referenceFasta
+        File referenceGtfFile
+        String shinyDir = "."
+
+        Int threads = 1
+        String memory = "10G"
+        Int timeMinutes = 90
+        String dockerImage = "quay.io/biocontainers/predex:0.9.1--pyh3252c3a_0"
+    }
+
+    command {
+        set -e
+        mkdir -p ${shinyDir}
+        predex annotation \
+            -f ${referenceFasta} \
+            -g ${referenceGtfFile} \
+            -o ${shinyDir}
+    }
+
+    output {        
+        File dgeAnnotation = shinyDir + "/annotation.tsv"
+    }
+
+    runtime {
+        cpu: threads
+        memory: memory
+        time_minutes: timeMinutes
+        docker: dockerImage
+    }
+
+    parameter_meta {
+        referenceFasta: {description: "The reference Fasta file.", category: "required"}
+        referenceGtfFile: {description: "The reference GTF file.", category: "required"}
+        shinyDir: {description: "The directory to write the output to.", category: "required"}
+        
+        threads: {description: "The number of threads to use.", category: "advanced"}
+        memory: {description: "The amount of memory this job will use.", category: "advanced"}
+        timeMinutes: {description: "The maximum amount of time the job will run in minutes.", category: "advanced"}
+        dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.",
+                      category: "advanced"}
```

Comment:
```suggestion
    parameter_meta {
        # inputs
        referenceFasta: {description: "The reference Fasta file.", category: "required"}
        referenceGtfFile: {description: "The reference GTF file.", category: "required"}
        shinyDir: {description: "The directory to write the output to.", category: "required"}
        threads: {description: "The number of threads to use.", category: "advanced"}
        memory: {description: "The amount of memory this job will use.", category: "advanced"}
        timeMinutes: {description: "The maximum amount of time the job will run in minutes.", category: "advanced"}
        dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.", category: "advanced"}
        
        # outputs
        dgeAnnotation: {description: ""}
```